gateway/src/apicast/policy/logging/readme.md

```diff
@@ -0,0 +1,172 @@
+header in the request, or `{{res.headers.FOO}}` to retrieve FOO header on
+response.
+- Service information, as `{{service.id}}` and all service properties as the
+  `THREESCALE_CONFIG_FILE` or `THREESCALE_PORTAL_ENDPOINT` parameters provide.
+
+## Examples
+
+### Disable access log:
+
+```json
+{
+  "name": "apicast.policy.logging",
+  "configuration": {
+    "enable_access_logs": false
+  }
+}
+```
+
+### Enable custom access log:
+
+```json
+{
+  "name": "apicast.policy.logging",
+  "configuration": {
+    "enable_access_logs": false,
+    "custom_logging": "[{{time_local}}] {{host}}:{{server_port}} {{remote_addr}}:{{remote_port}} \"{{request}}\" {{status}} {{body_bytes_sent}} ({{request_time}}) {{post_action_impact}}",
+  }
+}
+```
+
+### Enable custom access log with the service ID:
+```json
+{
+  "name": "apicast.policy.logging",
+  "configuration": {
+    "enable_access_logs": false,
+    "custom_logging": "\"{{request}}\" to service {{service.id}} and {{service.name}}",
+  }
+}
+```
+
+### Write access log in JSON format:
+
+```json
+{
+  "name": "apicast.policy.logging",
+  "configuration": {
+    "enable_access_logs": false,
+    "enable_json_logs": true,
+    "json_object_config": [
+      {
+        "key": "host",
+        "value": "{{host}}",
+        "value_type": "liquid"
+      },
+      {
+        "key": "time",
+        "value": "{{time_local}}",
+        "value_type": "liquid"
+      },
+      {
+        "key": "custom",
+        "value": "custom_method",
+        "value_type": "plain"
+      }
+    ]
+  }
+}
+```
+
+### Write a custom access log only for a successful request.
+
+```json
+{
+  "name": "apicast.policy.logging",
+  "configuration": {
+    "enable_access_logs": false,
+    "custom_logging": "\"{{request}}\" to service {{service.id}} and {{service.name}}",
+    "condition": {
+      "operations": [
+        {"op": "==", "match": "{{status}}", "match_type": "liquid", "value": "200"}
+      ],
+      "combine_op": "and"
+    }
+  }
+}
+```
+
+### Write a custom access log where reponse status match 200 or 500.
+
+```json
+{
+  "name": "apicast.policy.logging",
+  "configuration": {
+    "enable_access_logs": false,
+    "custom_logging": "\"{{request}}\" to service {{service.id}} and {{service.name}}",
+    "condition": {
+      "operations": [
+        {"op": "==", "match": "{{status}}", "match_type": "liquid", "value": "200"},
+        {"op": "==", "match": "{{status}}", "match_type": "liquid", "value": "500"}
+      ],
+      "combine_op": "or"
+    }
+  }
+}
+```
+
+## Caveats
+
+- If `custom_logging` or `enable_json_logs` property is enabled, default access
+  log will be dissabled.
+- If `enable_json_logs` is enabled, `custom_logging` field will be omitted
+
+## Global configuration for all services. 
+
+Logging options can be useful in all services, to avoid having issues with logs
+that are not correctly formated in other services, a custom Apicast environment
```

Comment:
`that are not correctly formated in other services, a custom APIcast environment`